lib/spack/spack/cmd/buildcache.py

```diff
@@ -299,8 +312,9 @@ def match_downloaded_specs(pkgs, allow_multiple_matches=False, force=False):
     return specs_from_cli
 
 
-def _createtarball(env, spec_yaml, packages, directory, key, no_deps, force,
-                   rel, unsigned, allow_root, no_rebuild_index):
+def _createtarball(env, spec_yaml, packages, output_location, key,
+                   no_deps, force, rel, unsigned, allow_root,
+                   no_rebuild_index):
```

Comment:
In a future PR we might want to refactor functions like this to avoid 11 arguments.

Reply:
👍